docs/README.md

```diff
@@ -1,33 +1,56 @@
 # Laravel Doctrine JSON:API
-Package for the [Laravel](https://laravel.com/) that allows developers to create [JSON:API](https://jsonapi.org/)
-endpoints using the [Doctrine ORM](https://www.doctrine-project.org/) for data persistence.
+Implement feature-rich [JSON:API](https://jsonapi.org/) compliant APIs
+in your [Laravel](https://laravel.com/) applications using [Doctrine ORM](https://www.doctrine-project.org/).
 
-It provides an easy-to-use API for building JSON:API responses and supports various features such as resource filtering,
-sorting, pagination, and relationships. With this library, Laravel developers can quickly implement a JSON:API compliant
-backend for their web or mobile applications.
+## Features
+- Built for Doctrine ORM
+- Battle-tested
+- Standardised, consistent APIs
+- Fetch resources
+- Fetch relationships
+- Inclusion of related resources (compound documents)
+- Sparse field sets
+- Sorting
+- Pagination
+- Filtering
+- Create resources
+- Update resources
+- Update relationships
+- Delete resources
 
-### Setup
-Please follow the [instructions](./Installation.md) and set up the package in your Laravel installation.
+## Installation & Config
+1. [Laravel](./Installation.md#laravel)
+2. [Doctrine Setup](./Installation.md#doctrine-setup)
```

Comment:
The link must be to `./LaravelDoctrine.md`

Reply:
It was right but the pages hierarchy was confusing. I improved it.